pom.xml

```diff
@@ -60,6 +60,7 @@
         <dep.jakarta.mail.version>2.0.0</dep.jakarta.mail.version>
         <dep.jeasy.version>4.1.0</dep.jeasy.version>
         <dep.jetty.version>11.0.15</dep.jetty.version>
+        <dep.logback.version>1.4.14</dep.logback.version>
```

Comment:
This commit includes two changes, dependency declaration and bump (1.4.8 -> 1.4.14). Please split into two commits.

Reply:
done

Reply:
The commit order is wrong. It should be "Make ... with 1.4.8" and then "Bump ..."

Reply:
fixed

Reply:
The 1st commit should contain the following line:
```xml
        <dep.logback.version>1.4.8</dep.logback.version>
```

> Bumped logback version to 1.4.14

Please avoid the past tense in the commit title.

Reply:
ah I see.

Reply:
this line never existed tho (I think) before this PR

Reply:
I missed `dep.logback.version` exists in airbase. No need to declare in this module. I just removed the 2nd commit.